index.js

```diff
@@ -177,11 +170,11 @@ function applyAttributes(t, openingElement, componentName, sourceFileName, attri
     sourceFileName
     && !ignoredComponentFromOptions
     && (componentName || ignoredElement === false)
-    && openingElement.node.attributes.find(node => {
+    && !openingElement.node.attributes.find(node => {
       if (!node.name) return
       return node.name.name === sourceFileAttributeName
```

Comment:
Oh, you could use the `hasNodeNamed` helper here too, I think.

Reply:
(Sorry, I looked at this earlier but didn't write anything then, and then I forgot a moment ago)

Reply:
Ah sorry I missed that one! Thanks for the catch!